var/spack/repos/spack_repo/builtin/packages/py_metatomic_torch/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+
+from spack.package import *
+
+
+class PyMetatomicTorch(PythonPackage):
+    """Torchscript bindings for metatomic"""
+
+    homepage = "https://docs.metatensor.org/metatomic"
+    url = "https://github.com/metatensor/metatomic/releases/download/metatomic-torch-v0.1.2/metatomic_torch-0.1.2.tar.gz"
+    git = "https://github.com/metatensor/metatomic.git"
+    pypi = "metatomic-torch/metatomic-torch-0.1.2.tar.gz"
```

Comment:
It seems like this package contains no versions? Can we use PyPI instead of github for the download?